apis/infix-en_GB/extensions/jdk8/atrium-api-infix-en_GB-jdk8-jvm/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/jdk8/OptionalAssertionsSpec.kt

```diff
@@ -0,0 +1,37 @@
+package ch.tutteli.atrium.api.infix.en_GB.jdk8
+
+import ch.tutteli.atrium.api.infix.en_GB.Empty
+import ch.tutteli.atrium.api.infix.en_GB.Present
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.specs.feature0
+import ch.tutteli.atrium.specs.fun0
+import ch.tutteli.atrium.specs.fun1
+import ch.tutteli.atrium.specs.integration.OptionalAssertionsSpec
+import ch.tutteli.atrium.specs.notImplemented
+import ch.tutteli.atrium.specs.testutils.WithAsciiReporter
+import java.util.*
+
+class OptionalAssertionsSpec : OptionalAssertionsSpec(
+    isEmpty = fun0(Companion::toBeEmpty),
```

Comment:
please remove the `isEmpty =` (also in other cases) it is obvious enough IMO when I read `toBeEmpty`